draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1179,8 +1185,9 @@ following:
 
 ~~~~~~~~~~~
     {
-        "error_message": "'start' cannot be greater than 'end'",
-        "error_code": "not compliant",
+        "type": "https://example.net/get-entries-error"
+        "title": "not compliant",
+        "detail": "'start' cannot be greater than 'end'",
```

Comment:
Using URNs (a la ACME), "not compliant" would become a camelCase suffix to the "type" parameter.  i.e.,

```
{
    "type": "urn:ietf:params:ct:error:notCompliant",
    "detail": "'start' cannot be greater than 'end'"
}
```

A few other thoughts...
"notCompliant" seems unnecessarily generic.  For problems with request parameters, ACME uses "malformed" and RFC7807 suggests "validation-error".
RFC7807 has the concept of subproblems, permitting details of multiple similar problems to be conveyed.  ACME does this.  Using https://tools.ietf.org/html/draft-ietf-acme-acme-14#section-6.6.1 and https://tools.ietf.org/html/rfc7807#section-3 for inspiration, this example might look something like this...

```
{
    "type": "urn:ietf:params:ct:error:malformed",
    "detail": "The request message was malformed",
    "subproblems": [
        {
            "name": "start",
            "detail": "must not be greater than 'end'"
        },
        {
            "name": "end",
            "detail": "must not be less than 'start'"
        }
    ]
}
```

Reply:
(1) Switched to 'malformed' as suggested.
(2) Not sure the 'subproblems' structure makes a lot of sense: All but one of the errors would fall into the 'malformed' category (the exception being 'shutdown' which is a bad log state).